THANKS.txt

```diff
@@ -220,6 +220,7 @@ Andrew Knyazev, the original author of LOBPCG, for advice on and maintenance of
    sparse.linalg.lobpcg
 Angeline G. Burrell for implementing nan_policy in the circular statistics
 Michael Marien for contributing to scipy.stats.entropy
+Vicente Reyes-Puerta for the function cut_tree_balanced in scipy.cluster.hierarchy
```

Comment:
I think we've removed `THANKS.txt` from the repo in favor of a wiki-based solution to reduce merge conflicts, etc.

Reply:
Hi @tylerjereddy, many thanks. I already deleted `THANKS.txt` from my PR.

Reply:
Hi @tylerjereddy, I think the `continuous-integration/travis-ci/pr` check is broken, isn't it? It has nothing to do with my PR, right? Many thanks and best regards.

Reply:
Yes, you can ignore the `TestMinimumPhase` error in Travis CI--that is caused by NumPy pre-release wheels and we are aware of it being worked on upstream.